plugin-init.py

```diff
@@ -44,14 +44,17 @@ def rewrite_files(files, name, description):
         print(file)
         update_file_contents(file, search_text='Skeleton', replace_text=name.capitalize())
         update_file_contents(file, search_text='skeleton', replace_text=name.lower())
-        update_file_contents(file, search_text='skeleton', replace_text=name.lower())
         update_file_contents(file, search_text='description = \'description\'',
                              replace_text=f"description = '{description}'")
+        update_file_contents(file, search_text='Plugin documentation', replace_text=description)
 
 
 def rename_files(files, name):
     for file in files:
-        os.rename(file, file.replace('skeleton', name.lower()))
+        if 'Skeleton' in file:
+            os.rename(file, file.replace('Skeleton', name.capitalize()))
```

Comment:
Why this over having the markdown filename be `skeleton.md`?

Reply:
Clearly figured writing 4 lines of Python was easier than one `mv`